lms/djangoapps/oauth_dispatch/urls.py

```diff
@@ -0,0 +1,21 @@
+"""
+OAuth2 wrapper urls
+"""
+
+from django.conf.urls import patterns, url
+from django.views.decorators.csrf import csrf_exempt
+
+from . import views
+
+
+urlpatterns = patterns(
+    '',
+    # authorize/ URL (below) not yet supported for DOT (MA-2124)
```

Comment:
Nit: Prefix this comment with a `TODO` so it comes up in our search for TODOs.  Good that there's a ticket number attached to it.